src/test/java/de/zalando/ep/zalenium/it/ParallelIT.java

```diff
@@ -115,7 +115,7 @@ private WebDriver getWebDriver() {
     public void checkIframeLinksForLivePreview(String browserType, Platform platform) {
 
         // Go to the homepage
-        getWebDriver().get(String.format("http://%s:%s/grid/admin/live", ZALENIUM_HOST, ZALENIUM_PORT));
+        getWebDriver().get(String.format("http://%s:%s/grid/admin/live", hostIpAddress, ZALENIUM_PORT));
```

Comment:
It could work in kubernetes in theory, if you run the test as a sidecar container (but it'd be fiddly), but just curious why we can't rely on `ZALENIUM_HOST` having the `hostIpAddress`? That way it can work for both docker and kubernetes.

What do we get out of finding the grid by using the current IP address inside the test? All it seems to do is ensure that maven is running on the same IP address as zalenium?

Reply:
The purpose of it was to check that the live view was reachable also by its ip and that the headers were being forwarded properly, since we broke that once. 

When the live view is done again, we can just do it from scratch.